qiskit_sphinx_theme/layout.html

```diff
@@ -267,116 +146,23 @@
-           };
-       </script>
-       {%- for scriptfile in script_files %}
-         <script type="text/javascript" src="{{ pathto(scriptfile, 1) }}"></script>
-       {%- endfor %}
-     {% endif %}
-
-  {% endif %}
-
-  <script type="text/javascript" src="{{ pathto('_static/js/vendor/popper.min.js', 1) }}"></script>
-  <script type="text/javascript" src="{{ pathto('_static/js/vendor/bootstrap.min.js', 1) }}"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/list.js/1.5.0/list.min.js"></script>
-  <script type="text/javascript" src="{{ pathto('_static/js/theme.js', 1) }}"></script>
-
-  <script type="text/javascript">
-      jQuery(function () {
-          SphinxRtdTheme.Navigation.enable({{ 'true' if theme_sticky_navigation|tobool else 'false' }});
-      });
-  </script>
-
-{%- block footer %} {% endblock %}
-
-  <div>
-    <br>
-  </div>
-
-  <!-- Begin Mobile Menu -->
-
-  <div class="mobile-main-menu">
-    <div class="container-fluid">
-      <div class="container">
-        <div class="mobile-main-menu-header-container">
-          <a class="header-logo" href="{{ theme_variables.external_urls['home'] }}" aria-label="PyTorch"></a>
-          <a class="main-menu-close-button" href="#" data-behavior="close-mobile-menu"></a>
-        </div>
-      </div>
-    </div>
-
-    <div class="mobile-main-menu-links-container">
-      <div class="main-menu">
-        <ul>
-          <li>
-            <a href="{{ pathto('getting_started') }}">Getting Started</a>
-          </li>
-          <li>
-            <a href="{{ pathto('tutorials') }}">Tutorials</a>
-          </li>
-          <li>
-            <a href="{{ theme_variables.external_urls['partners'] }}">Partners</a>
-          </li>
-          <br>
-          <li class="resources-mobile-menu-title">
-            Applications
-          </li>
-          <ul class="resources-mobile-menu-items">
-            <li>
-              <a href="{{ theme_variables.external_urls['ml'] }}">Machine learning</a>
-            </li>
-
-            <li>
-              <a href="{{ theme_variables.external_urls['nature'] }}">Nature</a>
-            </li>
-
-            <li>
-              <a href="{{ theme_variables.external_urls['finance'] }}">Finance</a>
-            </li>
-
-            <li>
-              <a href="{{ theme_variables.external_urls['optim'] }}">Optimization</a>
-            </li>
-          </ul>
-          <br>
-          <li>
-            <a href="{{ theme_variables.external_urls['experiments'] }}">Experiments</a>
-          </li>
-          <br>
-          <li class="resources-mobile-menu-title">
-            Resources
-          </li>
-
-          <ul class="resources-mobile-menu-items">
-            <li>
-              <a href="{{ theme_variables.external_urls['slack'] }}">Slack support</a>
-            </li>
-          </ul>
-          <br>
-        </ul>
-      </div>
-    </div>
-  </div>
-
-  <!-- End Mobile Menu -->
```

Comment:
Trying to understand this removal. Were these lines removed because we replace the old top menu with the new mega nav menu? And the new mobile menu will behave the same as [qiskit.org](https://qiskit.org/)?